### ISSUE TITLE
Port to modules

### ISSUE THREAD

**Issue by breeze4** [vonWolfehaus/von-grid#23]
Hey there, awesome library @vonWolfehaus! Thank you for implementing the hex logic and Three.JS is a great choice.

In case anyone coming across this repo wants to use it as a standard module like I am trying to, I ported it over to modules and away from a global variable to make it dead-simple to use in projects with module loading already setup (I'm using it in a `create-react-app` app, so it needed to be in modules): https://github.com/breeze4/von-grid

The diff: https://github.com/vonWolfehaus/von-grid/compare/master...breeze4:master

Basically I ran a code formatter over it, turned all the `vg.WhateverObject` into importing `WhateverObject`, moved the constants into a separate file and export the things an end-user would need.

I haven't fully used the entire thing, so there might be some problems, but so far it works really well. I'll probably find problems with it as I keep using it and will try to fix issues as I find them.

Hopefully someone else finds this useful.

**Comment by stlbucket** [comment]
@breeze4 did you publish an npm package?

**Comment by breeze4** [comment]
@stlbucket I did not publish an npm package.

After posting this, I am making more changes to it, such as adding (optional) support for shadows and swapping the camera controls.

I'll close this issue for now because I don't intend to publish the forked changes until I've gotten it to where I'm happy with the updated functionality. Maybe there will be a path to re-incorporating those changes back upstream.
